packages/idempotency/src/index.ts

```diff
@@ -2,3 +2,5 @@ export * from './Exceptions';
 export * from './IdempotencyConfig';
 export * from './idempotentDecorator';
 export * from './makeFunctionIdempotent';
+export * from './middleware';
```

Comment:
See comment I left under `packages/idempotency/src/persistence/index.ts` about imports/exports - TL;DR; the omission was intentional and I believe should stay like that.